Index: NetworkManager-0.6.6/src/nm-dbus-net.c
===================================================================
--- NetworkManager-0.6.6.orig/src/nm-dbus-net.c
+++ NetworkManager-0.6.6/src/nm-dbus-net.c
@@ -70,6 +70,9 @@ static NMAccessPoint *nm_dbus_get_ap_fro
 		compare_path = g_strdup_printf ("%s/%s/Networks/%s",
 		                                NM_DBUS_PATH_DEVICES, escaped_dev, escaped_ssid);
 
+		g_free(escaped_dev);
+		g_free(escaped_ssid);
+
 		len = strlen (compare_path);
 		if (strncmp (path, compare_path, len) == 0) {
 			/* Differentiate between 'foo' and 'foo-a' */
