--- configure	2009-09-05 17:33:56.000000000 +0200
+++ configure.new	2009-09-19 16:46:10.966008667 +0200
@@ -59,8 +59,9 @@
 	    exit 0
 	    ;;
 	* )
-	    usage
-	    exit 1
+	    #The next two lines were commented out by the Gentoo ebuild to allow the default flags from econf:
+	    #usage
+	    #exit 1
 	    ;;
     esac
     shift
@@ -282,7 +283,7 @@
 	FPP=-fpp
 	;;
     ifort)
-	FFLAGS="$FFLAGS"
+	FFLAGS="$FFLAGS -vec-report0"
 	FPP=-fpp
 	DEB="-g -check all"
 	;;
