--- Makefile	2009-05-16 18:07:01.000000000 -0500
+++ Makefile.new	2009-05-16 20:12:11.000000000 -0500
@@ -11,31 +11,31 @@
 endif
 
 test : lib
-	@cd tests; $(MAKE) clean; $(MAKE) -j 8 run
+	cd tests; $(MAKE) clean; $(MAKE) -j 8 run
 
 
 lib : libfgsl_$(F90).a 
 
 libfgsl_$(F90).a : fgsl.o fgsl_utils.o
-	@ar $(ARFLAGS) libfgsl_$(F90).a $^
+	ar $(ARFLAGS) libfgsl_$(F90).a $^
 	@echo "Build complete"
 
 fgsl.o : fgsl.f90 api/*.finc interface/*.finc
-	@$(F90) $(FFLAGS) -c  fgsl.f90
+	$(F90) $(FFLAGS) -c  fgsl.f90
 
 %.o : %.f90
-	@$(F90) $(FFLAGS) -c -o $@ $^
+	$(F90) $(FFLAGS) -c -o $@ $^
 
 %.o : %.c
-	@$(CC) -c $(GSL_INC) $(CFLAGS) -o $@ $^
+	$(CC) -c $(GSL_INC) $(CFLAGS) -o $@ $^
 
 install : lib
-	mkdir -p $(PREFIX)
-	cp -p libfgsl_$(F90).a $(PREFIX)/$(LIB)
-	chmod ugo+r $(PREFIX)/$(LIB)/libfgsl_$(F90).a
-	mkdir -p $(PREFIX)/include/$(F90)
-	cp -p fgsl.mod $(PREFIX)/include/$(F90)
-	chmod ugo+r $(PREFIX)/include/$(F90)/fgsl.mod
+	mkdir -p $(DESTDIR)$(VENDORPREFIX)/$(LIB)
+	cp -p libfgsl_$(F90).a $(DESTDIR)$(VENDORPREFIX)/$(LIB)
+	chmod ugo+r $(DESTDIR)$(VENDORPREFIX)/$(LIB)/libfgsl_$(F90).a
+	mkdir -p $(DESTDIR)$(VENDORPREFIX)/include
+	cp -p fgsl.mod $(DESTDIR)$(VENDORPREFIX)/include/fgsl_$(F90).mod
+	chmod ugo+r $(DESTDIR)$(VENDORPREFIX)/include/fgsl_$(F90).mod
 
 doc : 
 	cd doc; $(MAKE)
