--- configure	2008-08-31 20:37:23.000000000 +0200
+++ configure.new	2009-09-19 14:31:14.612919757 +0200
@@ -53,8 +53,9 @@
 	    exit 0
 	    ;;
 	* )
-	    usage
-	    exit 1
+	    # The next two lines were commented out by the Gentoo ebuild to allow the default flags from econf:
+	    #usage
+	    #exit 1
 	    ;;
     esac
     shift
@@ -266,6 +267,7 @@
 	FPP=-fpp
 	;;
     ifort)
+	FFLAGS="$FFLAGS -vec-report0"
 	FPP=-fpp
 	;;
     pgf90)
