diff -Naur tmp-old/configure tmp-new/configure
--- configure	2006-08-30 20:08:04.000000000 +0000
+++ configure	2006-08-31 03:01:38.000000000 +0000
@@ -3855,7 +3855,7 @@
    echo "$as_me:$LINENO: result: $gcc_version" >&5
 echo "${ECHO_T}$gcc_version" >&6
 
-   DEST_LIBDIR=$prefix/lib/gcc-lib/$host/$gcc_version
+   DEST_LIBDIR=$prefix/lib/gcc-lib/$host/$gcc_version/
    SAVE_DIR=`pwd`
    PREFIX=$prefix
 
@@ -4397,7 +4397,7 @@
 case $host in
   *x86_64* | *amd64* | *powerpc64* )
     if test -d /usr/lib64 ; then
-       STARTFILE_PREFIX_2=/usr/lib64
+       STARTFILE_PREFIX_2=/usr/lib64/
     fi
     ;;
 
